Assets/Python/RFCUtils.py

```diff
@@ -1573,25 +1572,26 @@ def canRespawn(self, iPlayer):
 			for tInterval in tResurrectionIntervals[iPlayer]:
 				iStart, iEnd = tInterval
 				if getTurnForYear(iStart) <= iGameTurn <= getTurnForYear(iEnd):
-					bPossible = True
 					break
+			else:

```

Comment:
Yes. Python has a else clause for for loops.
http://book.pythontips.com/en/latest/for_-_else.html

The else activates when the for loops completes normally. If the loop is terminated with a break statement, the else clause does not activate.